hypervisor/arch/x86/guest/instr_emul.c

```diff
@@ -1312,7 +1312,8 @@ emulate_stack_op(struct vcpu *vcpu, uint64_t mmio_gpa, struct vie *vie,
 		pr_err("TODO: inject ss exception");
 	}
 
-	if (vie_alignment_check(paging->cpl, size, cr0, rflags, stack_gla) != 0) {
+	if (vie_alignment_check(paging->cpl, size, cr0, rflags, stack_gla)
```

Comment:
Do you expect the  API return bool? But it is not,

Reply:
vie_alignment_check will return either 0 or 1.
What I changed here is to make sure that the line length is less than 80.